app/views/layouts/application.html.erb

```diff
@@ -75,7 +75,9 @@
             <ul class="nav nav-list">
               <li><%= link_to "Conversations", "/conversations"  %></li>
               <li><%= link_to "People", "/people"  %></li>
-              <li><%= link_to "Edit Users", users_path %></li>
+              <% if current_user && current_user.role == "manager" %>
```

Comment:
Can we use the `.manager?` method provided by the `User` model to push this logic down to the model?